module/programUsers/helper.js

```diff
@@ -5,160 +5,169 @@
-                if (fieldsArray != "all") {
-                    fieldsArray.forEach(field => {
-                        projection[field] = 1;
-                    });
-                }
-               
-                if( skipFields !== "none" ) {
-                   skipFields.forEach(field=>{
-                       projection[field] = 0;
-                   });
-                }
-                let programUsers;
-                if( sortedData !== "" ) {       
-                    programUsers = await database.models.programUsers
-                    .find(queryObject, projection)
-                    .sort(sortedData)
-                    .lean();
-                } else {
-                    programUsers = await database.models.programUsers
-                    .find(queryObject, projection)
-                    .lean();
-                } 
-           
-                return resolve(programUsers);
-            
-            } catch (error) {
-                return reject(error);
-            }
-        });
-    }
-    
-    /**
-     * check if user joined a program or not and consentShared
-     * @method
-     * @name checkForUserJoinedProgramAndConsentShared
-     * @param {String} programId
-     * @param {String} userId 
-     * @returns {Object} result.
-    */
+        //Check data present in programUsers collection.
+        let programUsers = await this.programUsersDocuments(query, [
+          "_id",
+          "consentShared",
+        ]);
+        result.joinProgram = programUsers.length > 0 ? true : false;
+        result.consentShared =
+          programUsers.length > 0 ? programUsers[0].consentShared : false;
+        return resolve(result);
+      } catch (error) {
+        return reject(error);
+      }
+    });
+  }
 
-    static checkForUserJoinedProgramAndConsentShared(
-        programId, 
-        userId
-    ) {
-        return new Promise(async (resolve, reject) => {
-            try {
-                let result = {};
-                const query = { 
-                    userId: userId,
-                    programId: programId
-                }
-    
-                //Check data present in programUsers collection.
-                let programUsers = await this.programUsersDocuments(
-                    query,
-                    ["_id","consentShared"]
-                );
-                result.joinProgram = programUsers.length > 0 ? true : false;
-                result.consentShared = programUsers.length > 0 ? programUsers[0].consentShared : false;
-                return resolve(result);
-            
-            } catch (error) {
-                return reject(error);
-            }
-        });
-    }
-}
\ No newline at end of file
+  /**
+   * bulkUpdate function is used to do bulk operations.
+   * @method
+   * @name bulkUpdate
+   * @param {Object} [Data] -  Data along with filter and quries.
+   * @returns {Object} - status of bulk operation.
+   */
+  static bulkUpdate(Data) {
```

Comment:
@ankitpws - This is not required

Reply:
removed this function instead of this we are using updateMany